connector/src/test/scala/com/microsoft/kusto/spark/KustoClientTests.scala

```diff
@@ -15,7 +15,8 @@ import scala.collection.JavaConverters._
 
 @RunWith(classOf[JUnitRunner])
 class KustoClientTests extends FlatSpec  with Matchers{
-  class KustoClientStub (override  val clusterAlias: String, override  val engineKcsb: ConnectionStringBuilder, override val ingestKcsb: ConnectionStringBuilder, var tagsToReturn: util.ArrayList[String]) extends KustoClient(clusterAlias, engineKcsb, ingestKcsb){
+  class KustoClientStub (override  val clusterAlias: String, override  val engineKcsb: ConnectionStringBuilder, override val ingestKcsb: ConnectionStringBuilder, var tagsToReturn: util.ArrayList[String])
+    extends KustoClient(engineKcsb, ingestKcsb, clusterAlias){
```

Comment:
Is this standard to put the `extends` on a new line? It might be NBD, but I'm concerned it might make it harder to search the codebase.

Reply:
not really its actually used in new line only if its after last input variable

Reply:
I think it's better to keep it on the same line to make it easier to find, but your call.

Reply:
yea i fixed